ref/general/authentication.adoc

```diff
@@ -17,54 +17,57 @@ In order for an application to be secure, it must be able to determine who its u
 
-For example, a user of a music streaming service might want to listen to a song, learn about the artist, and get recommendations for similar artists. Each of these tasks would require access to a different service, but having to authenticate for each of them separately uses unnecessary time and resources. Open Liberty supports a number of different strategies for single sign-on (SSO) authentication so that a user or service must authenticate only once to access the various resources in an application. For example, the link:/docs/ref/feature/#socialLogin-1.0.html[Social Media Login] feature enables users of an application that is running on Open Liberty to authenticate through their social media login. After authentication, an application can either obtain a JSON Web Token (JWT) from the social media service or associated OpenID Connect provider, or create a new one. This JWT can then be used to authenticate the user to any service in the application.
+For example, a user of a streaming music service might want to listen to a song, learn about the artist, and get recommendations for similar artists. Each of these tasks would require access to a different service, but authenticating to each service separately uses unnecessary time and resources. Open Liberty supports a number of different strategies for single sign-on (SSO) authentication so that a user or service authenticates only one time to access the various resources in an application. For example, the link:/docs/ref/feature/#socialLogin-1.0.html[Social Media Login] feature enables users of an application that is running on Open Liberty to authenticate through their social media login. After authentication, an application can either obtain a JSON Web Token (JWT) from the social media service or associated OpenID Connect provider, or create a new one. This JWT can then authenticate the user to any service in the application.
+
+In the following diagram, a user requests access to protected resources in an application. During authentication (steps 1-4), the user’s request is routed to an Open ID Connect identity provider, which confirms the user's identity and security roles. Then identity provider sends authentication data back to the application, where it is used to authorize user access to protected resources. During authorization, Open Liberty checks the roles that are specified for the user against the roles that are defined for the application resources. The user is granted access to any resources that match their role.
```

Comment:
In 'During authentication (steps 1-4)', avoid the parentheses.

'Then identity provider sends authentication data back to the application, '
Add a comma after "Then"